crates/modelardb_server/src/optimizer/model_simple_aggregates.rs

```diff
@@ -12,6 +12,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+//! Implementation of a physical optimizer rule that rewrites aggregates that are computed from
+//! reconstructed values from a single column without filtering so they are computed directly from
+//! segments instead of the reconstructed values.
```

Comment:
Maybe remove `of the reconstructed values`.

Reply:
Fixed.